### ISSUE TITLE
Build failure: fh

### ISSUE THREAD

**Issue by MisileLab** [NixOS/nixpkgs#331240]
### Steps To Reproduce

1. build `fh`

### Build log

[hydra can reproduce](https://hydra.nixos.org/build/267233599)

### Notify maintainers

@figsoda 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.10.2, NixOS, 24.11 (Vicuna), 24.11.20240731.c841bb4`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/bah149sa1nz43q8iwa042v5chlh4hqqv-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by modisek** [comment]
I am having a simillar problem currently

**Comment by ghpzin** [comment]
@modisek
That issue was fixed with update PR - #331327
Currently builds on hydra on 4 main platforms are green (on both `nixos-unstable` and `nixos-24.11`), so unless you use pinned nixpkgs that old it should not be related.
If you still see build failure on current channel, I would recommend creating new issue with logs.
